src/KubernetesClient/ModelExtensions.cs

```diff
@@ -0,0 +1,356 @@
+using System;
```

Comment:
This code looks fine, but it's hard to read in places because all of the combining of `if/else` and the code on the same line.

Please use explicit curly braces for all if/else. I realize it makes the code more verbose, but it also makes it significantly easier to read, and since this is a community library, readability is important. Thanks!

Reply:
I wrapped all the if statements.